content/en/tracing/trace_explorer/trace_view.md

```diff
@@ -21,23 +21,85 @@ further_reading:
+
+
+{{% /tab %}}
+{{% tab "Span List" %}}
+
+{{< img src="tracing/trace_view/spanlist.png" alt="Trace View" style="width:90%;">}}
+
+Displays [resources][1] by grouping ([service][2] by default) and sorts them according to their count of spans. This visualization is useful for scanning latency information by resource or grouping.
+
+Filter resources by type or naming information using the corresponding buttons and text-based search.
+
+{{< img src="tracing/trace_view/spanlist_headers.png" alt="Span List headers" style="width:90%;">}}
+
+Sort groupings by clicking on the corresponding column header: **RESOURCE**, **SPANS**, average duration (**AVG DURATION**), execution time (**EXEC TIME**), or percentage of trace execution time (**% EXEC TIME**).
+
+[1]: /tracing/glossary/#resources
+[2]: /tracing/glossary/#services
+{{% /tab %}}
+{{% tab "Waterfall (Beta)" %}}
+
+<div class="alert alert-info"><strong>Join the Beta!</strong><br />
+To join the Waterfall private beta <a href="https://forms.gle/LjJR1ZbF1tNDv5JC6">fill out this form</a>.</div>
+
+{{< img src="tracing/trace_view/waterfall.png" alt="Waterfall" style="width:90%;">}}
+
+Displays all the spans for an associated trace, color-coded on separate rows and on a timeline. This visualization is useful for isolating relevant parts of a trace and understanding how asynchronous processes fit into the execution flow.
+
+On each row (that is, per span):
+* A bar (colored by service), whose length corresponds to the percentage of total trace duration
+* The service name, operation name, and resource name, with font styling: **service** operation <span style="color:gray">resource</span>
+* Absolute and relative span duration information
+* (when applicable) An error icon or HTTP status code
+
+To expand or collapse span descendants, click the plus or minus button on any row. To expand all spans, click the plus button to the left of the timescale.
+
+{{% /tab %}}
+{{% tab "Map" %}}
+
+{{< img src="tracing/trace_view/map.png" alt="Map" style="width:90%;">}}
+
+Displays a service-level representation of the trace, with arrows showing the flow of calls between services. This visualization is useful for getting a high-level overview of the trace and the order in which services are called.
+
+Hover over a service to highlight its parent and children, and click on it to focus on the service entry span.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## More information
+
+The height-adjustable bottom of the Trace View shows selected span and trace information. 
+
+The span header contains service, operation, and resource names of the selected span as well as latency information. Pivot to other parts of the platform or narrow down your [Trace Explorer][5] search by clicking on the naming pill.
 
-### More information
+{{< img src="tracing/trace_view/span_header.png" alt="Span header" style="width:90%;">}}
 
 {{< tabs >}}
```

Comment:
The screenshots of this section are all super old. @lucasdmaley can you update them and do a quick check on the text at the same time?

Reply:
Resolved with most recent commit